src/components/Cards.jsx

```diff
@@ -0,0 +1,87 @@
+import { React, useState, useEffect } from "react";
+import BackgroundUrls from "./BackgroundUrls";
+
+const Cards = () => {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    fetch(`https://swapi.dev/api/films`)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`Status is ${response.status}`);
+        }
+        return response.json();
+      })
+      .then((rawData) => {
+        console.log(rawData);
+        setData(rawData.results);
```

Comment:
let duplicate = [...rawData.results]

const newResults = duplicate.map((item, index) => item.imageUrl = BackgroundUrls[index])

setData(newResults)

Reply:
Ok, Thank you

Reply:
Done, though setData(newResults) rendered only the images, while debugging, I found out setData(duplicate) worked. 

Is it ok to merge, now?